react-components/stories/HighlightNode.stories.tsx

```diff
@@ -127,17 +127,21 @@ const StoryContent = ({ resources }: { resources: AddResourceOptions[] }): React
       ]);
     } else if (nodeData?.intersection !== undefined && 'annotation' in nodeData.intersection) {
       const assetId =
-        (nodeData.intersection.annotation.annotation.data as AnnotationsBoundingVolume).assetRef
-          ?.externalId ??
-        (nodeData.intersection.annotation.annotation.data as AnnotationsBoundingVolume).assetRef
-          ?.id;
+        (
+          nodeData.intersection.annotation.annotation
+            .data as AnnotationsCogniteAnnotationTypesImagesAssetLink
+        ).assetRef?.externalId ??
+        (
+          nodeData.intersection.annotation.annotation
+            .data as AnnotationsCogniteAnnotationTypesImagesAssetLink
+        ).assetRef?.id;
```

Comment:
Can we refactor this expression by putting the common part in a constant? :sweat_smile: